serpentTools/objects/xsdata.py

```diff
@@ -0,0 +1,120 @@
+"""Holds cross section data pertaining to Serpent xsplot output."""
+import numpy as np
+from matplotlib import pyplot
+
+from serpentTools import messages
+from serpentTools.objects import NamedObject, convertVariableName
+
+
+class XSData(NamedObject):
+    docParams = """name: str
+        Name of this material
+    metadata: dict
+        Dictionary with file metadata"""
+    __doc__ = """
+    Base class for storing cross section data an xsplot file
+
+    Parameters
+    ----------
+    {params:s}
+
+    Attributes
+    ----------
+
+    """.format(params=docParams)
+
+    def __init__(self, name, metadata, isIso=False):
+        NamedObject.__init__(self, name)
+
+        # Whether this describes individual isotope XS, or whole-material XS
+        self.isIso = True
+
+        # serpent starts their names with an "m" if it's a whole-material XS
+        self.isIso = isIso
+
+        # energy grid for the nuclides
+        self.egrid = None
+
+        # metadata reference
+        self.metadata = metadata
+
+        # Possible reactions on this material / nuclide
+        # Maps MT integer number to a description
+        self.MT = []
+        self.MTdescrip = []
+
+        # Holds XS numeric data
+        self.xsdata = None
+
+        # whether nu data present for fissionables
+        self.hasNuData = False
+
+    @staticmethod
+    def negativeMTDescription(mt):
```

Comment:
Could this instead just be a class attribute, i.e.
```
class XSData(----)
    NEGATIVE_MTS = {}
```
Saves the need to rebuild/redeclare this every time the method is called.

Reply:
The code is 110% cleaner due to this suggestion. thanks.